python/ray/experimental/dag/tests/test_function_dag.py

```diff
@@ -170,6 +170,42 @@ def d(nested):
     assert ray.get(ct.get.remote()) == 4
 
 
+def test_dag_options(shared_ray_instance):
```

Comment:
Could you also cover options overwriting?

Reply:
do you mean this?

```python
    assert Foo.options(num_gpus=100).bind().get_options() == {"num_gpus": 100}
```

we already have this right?

Reply:
sure